### PR TITLE
Implementing Acl Sync

### PR DESCRIPTION
**Pull request by naemono**

### CODE REVIEW

**Comment by naemono** [comment]
Python3 is failing here.  I'll fix.

**Comment by naemono** [comment]
Py3.x Fixed!

File: objectrocket/bases.py [hunk 1]
```diff
@@ -126,6 +127,52 @@ def __repr__(self):
...
+            resp_json = response.json()
+            current_status = resp_json.get('data', {})
+            data.update({"aws_acl_sync_enabled": current_status.get("aws_acl_sync_enabled", False),
+                         "rackspace_acl_sync_enabled": current_status.get("rackspace_acl_sync_enabled", False)})
+            if aws_sync is not None:
+                data.update({"aws_acl_sync_enabled": aws_sync})
+            if rackspace_sync is not None:
+                data.update({"rackspace_acl_sync_enabled": rackspace_sync})
+
+            response = requests.put(url, data=json.dumps(data), **self._instances._default_request_kwargs)
```
Comment (TheDodd) [inline objectrocket/bases.py]: You can also say `json=data` and the requests lib will JSON encode the dict for you. Convenient, but not critical.

File: tests/test_auth.py [hunk 2]
```diff
@@ -1,33 +1,62 @@
...
+import json
 import mock
 import pytest
+import responses
 
 from objectrocket import errors
 from objectrocket.auth import Auth
-
+from objectrocket.client import Client
+
```
Comment (TheDodd) [inline tests/test_auth.py]: Need a second blank line above this, else linter error.

Comment (TheDodd) [inline tests/test_auth.py]: Strange that the linter is not complaining about this.

**Comment by TheDodd** [comment]
Have a linter PR pointed to this PR for a merge. Will expose some static code issues which the linter was ignoring (it was technically ignoring everything). **My misconfiguration of the linter from a while back, to be sure.**

**Comment by TheDodd** [comment]
+1 will address linter issues in separate PR. Merge at will.

File: tests/test_auth.py (hunk 2 above)
Comment (naemono) [inline tests/test_auth.py]: Strange indeed.  Line added.

File: objectrocket/bases.py (hunk 1 above)
Comment (naemono) [inline objectrocket/bases.py]: This is nice to know... Removed that import and adjusted those calls.

**Pull request merged**